Wikipedia/Code/MWKDataStore.m

```diff
@@ -459,6 +457,12 @@ - (void)performUpdatesFromLibraryVersion:(NSUInteger)currentLibraryVersion inMan
             DDLogError(@"Error saving during migration: %@", migrationError);
             return;
         }
+        
+        [self moveImageControllerCacheFolderWithError:&migrationError];
```

Comment:
@joewalsh - do we increment the library version for each type of migration, i.e. multiple per release? Or should we add this change and the standard user defaults change into version 9?

Reply:
@tonisevener depends - usually if something has gone to public beta we'll increment the library version to ensure everyone's data gets updated. Since this is just in internal beta, I was OK with leaving it with the previous migration (and having internal beta testers re-install the app)